Dockerfile

```diff
@@ -113,6 +113,7 @@ COPY --from=builder /usr/local/lib/python3.10/dist-packages/ /usr/local/lib/pyth
 COPY --from=builder /usr/local/bin/ns* /usr/local/bin/
 
 # Install nerfstudio cli auto completion
+WORKDIR /workspace
```

Comment:
Small change, otherwise I had some directory issue running the next command.

Reply:
I don't think this should be needed...

Reply:
Shouldn't be needed with the last version.